icu4c/source/i18n/unicode/measunit.h

```diff
@@ -2595,7 +2596,8 @@ class U_I18N_API MeasureUnit: public UObject {
 
     /**
      * Returns by value, unit of mass: metric-ton.
-     * Also see {@link #createMetricTon()}.
+      * (renamed to tonne in CLDR 42 / ICU 72)
+    * Also see {@link #createMetricTon()}.
```

Comment:
nit: mis-indentation

Reply:
Thanks, will fix in a future PR for this release